common/courses/states.ts

```diff
@@ -70,12 +71,24 @@ export async function allowCourse(course: Course, screeningComment: string | nul
     // Usually when a new course is created, instructors also create a proper subcourse with it
     // and then forget to publish it after it was approved. Thus we just publish during approval,
     // assuming the subcourses are ready:
-    const subcourses = await prisma.subcourse.findMany({ where: { courseId: course.id } });
+    const subcourses = await prisma.subcourse.findMany({
+        where: { courseId: course.id },
+        include: { subcourse_instructors_student: { select: { student: true } } },
+    });
     for (const subcourse of subcourses) {
         if (await canPublish(subcourse)) {
             await publishSubcourse(subcourse);
         }
     }
+
+    const [subcourse] = subcourses;
+    subcourse.subcourse_instructors_student.forEach(async (instructor) => {
```

Comment:
I think it doesn't hurt to iterate over `subcourses`. This would at least ensure that it's not breaking when we want to support multiple subcourses for a course.